corehq/sql_accessors/migrations/0055_drop_forms_cases_by_id_accessors.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-09-19 15:48
+from __future__ import unicode_literals
+
+from django.db import migrations
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sql_accessors', '0054_drop_reindexa_accessor_functions'),
+    ]
+
+    operations = [
+        HqRunSQL("DROP FUNCTION IF EXISTS get_cases_by_id(TEXT[])"),
+        HqRunSQL("DROP FUNCTION IF EXISTS get_forms_by_id(TEXT[])"),
```

Comment:
This should be split up into a separate PR so that we don't drop a function that could be used by old webworkers

Reply:
makes sense, thanks.

Reply:
Just to confirm we have to split this out since migration happen while the old webworkers are running.